app/main/controller/image_ocr_controller.py

```diff
@@ -37,7 +38,7 @@ def post(self):
             return
```

Comment:
do we want to log "no text found in image using OCR" here so we can compare the number of misses?